### PR TITLE
Added toast.

### PR DESCRIPTION
**Pull request by bre1470**
Added ToastScript. 🍞

### CODE REVIEW

**Comment by bre1470** [comment]
Can someone please request a change for the typo?

File: toast.js [hunk 1]
```diff
@@ -0,0 +1 @@
+console.log('taostscript');
```
Comment (pawellysy) [inline toast.js:1]: TYPO!

**Review by pawellysy — CHANGES_REQUESTED** [review]
TYPO!!!!

**Review by pawellysy — APPROVED** [review]

**Comment by bre1470** [comment]
@pawellysy Thank you! Milestone assignment works. 👍

**Pull request merged**